src/main/java/com/faforever/client/fa/relay/GpgClientCommand.java

```diff
@@ -24,7 +24,7 @@
   REHOST("Rehost"),
   DESYNC("Desync"),
   GAME_FULL("GameFull"),
-  ENDED("Ended"),
+  GAME_ENDED("GameEnded"),//TODO: it's called GameEnded
```

Comment:
What needs to be done here, exactly? And comments should be put on the line above, not behind